artifacts/oc-certified/0.1.0/manifests/vdo-controller-manager-metrics-service-rbac.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: Service
```

Comment:
this is metrics-service. what are we naming it rbac

Reply:
this is already merged in the certified operators project so we cannot modify it, we will update this in next release